packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -268,11 +275,10 @@ class OnsetPicker extends React.Component {
 
     const onsetObject = {
       precision: this.state.precision,
-      granularity: this.state.granularity,
       onsetDate: this.state.onsetDate ? this.state.onsetDate.format(DATE_FORMAT) : '',
+      granularity: this.state.precision !== PrecisionOptions.UNKNOWN ? this.state.granularity : '',
```

Comment:
Granularity should always be undefined for Unknown at this point in the code. Is there ever a case during this check where this isn't true? 

This check should be unnecessary. If for passivity we want to return a string instead of undefined we can `or` the assignment. 

```jsx
granularity: this.state.granularity || '', // Granularity should always be undefined for Unknown precision
```

Reply:
When I first created the component I just didn't clear the non-precision values out when setting precision to Unknown. My assumption was that if you get an Unknown precision the other values are irrelevant. But it should be fine and is probably a good idea to just clear out non-precision values.

Reply:
in talking with @doogthemushroom he thinks we should be clearing the values on change.  it may have been something that was missed.